### PR TITLE
Add a warning to Introduction to Stylometry with Python (French and English); closes #1883

### LINKED ISSUE #1883: Potential Bug in Introduction to Stylometry in Python
A reader sent me a note about some difficulties she encountered when working through the lesson:

"While replicating we came across a few errors(?) in the code mentioned on the website; the outputs for Kilgariff's Chi-Squared Method and John Burrows' Delta Method do not match what has been stated.

We figured a probable error as to what was wrong with the Chi-Squared Method, but couldn't understand the bug in Delta Method."

image(2)

She tried this to solve the issue:
image(3)

Tagging @fdlaramee to take a look.

### PR DESCRIPTION
**Pull request by fdlaramee**
A user had trouble reproducing the results in the tutorial because they had skipped one of the methods and moved straight to the second or third. I added a warning that users who want to do that must make sure to copy-paste the preprocessing code from the method(s) they skip if they want to obtain the same results that I do.

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Assign at least one individual or team to "Reviewers"
  - [ ] if the text needs to be translated, assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing edtor in your PR. Please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines) when writing your PR description
- [x] Add the appropriate "Label"
- [x] [Ensure the Travis CI checks pass](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#4-check-your-pr-status)
- [x] [Check the live preview of your PR on Netlify](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#5-preview-how-your-pr-looks-when-built-into-html)
- [x] If this PR closes an open issue, add the phrase `Closes #ISSUENUMBER` to the description above

*If you are having difficulty fixing Travis errors, first consult <https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions> carefully, especially ["Common Travis Errors"](https://github.com/programminghistorian/jekyll/wiki/Making-Technical-Contributions#common-travis-errors). Then contact the technical team if you need further help.*

### CODE REVIEW

**Comment by fdlaramee** [comment]
Hi @programminghistorian/technical-team ; I don't understand why this build fails; it reports broken links in three documents that I haven't changed; help please!

**Comment by walshbr** [comment]
Travis checks for URLs that are broken, and it checks the whole site - not just the pages you changed. So those are URLs that have gone down since we last built things. Even if you didn't change those pages, we generally have a policy that the person who finds the broken links fixes them as a part of the same PR. If they're not easily fixed because the resources can't be found in the past I think we've removed those links and said as much in a little emendation in the text.

**Comment by jenniferisasi** [comment]
> Hi @programminghistorian/technical-team ; I don't understand why this build fails; it reports broken links in three documents that I haven't changed; help please!

Every time we run anything, Travis looks for errors all over the site; so this is totally normal @fdlaramee :) 

It is weirder however that two of the links are actually working when I checked now.

**Comment by walshbr** [comment]
If that's the case you can click "re-run" next to the place where you click "build" to see the errors. I just reset. Sometimes things happen to be down temporarily when Travis runs.

**Comment by walshbr** [comment]
Things seem fine now. (and FWIW in case you went looking, it looks like the "re-run" button disappears when the travis checks pass - I swear it was there five minutes ago).

File: fr/lecons/introduction-a-la-stylometrie-avec-python.md [hunk 1]
```diff
@@ -52,6 +52,10 @@ L'identification de l'auteur d'un texte anonyme constitue cependant l'une des ap
 * Comment utiliser des structures de données relativement avancées, comme les [dictionnaires](https://fr.wikipedia.org/wiki/Dictionnaire_des_donn%C3%A9es) de [chaînes de caractères](https://fr.wikipedia.org/wiki/Cha%C3%AEne_de_caract%C3%A8res) et les dictionnaires de dictionnaires, dans le langage de programmation [Python](https://fr.wikipedia.org/wiki/Python_(langage)).
 * Les rudiments du [Natural Language Toolkit (documentation en anglais)](https://www.nltk.org/) (nltk), un module Python populaire dédié au [traitement du langage naturel](https://fr.wikipedia.org/wiki/Traitement_automatique_du_langage_naturel).
 
+<div class="alert alert-warning">
+Veuillez noter que le code informatique de cette leçon a été conçu pour être exécuté en séquence. Si, par exemple, vous désirez passer outre la méthode de Mendenhall et vous diriger directement vers celle de Klgariff ou celle de Burrows, veuillez vous assurer de copier-coller les instructions de pré-traitement qui apparaissent dans la description de la méthode de Mendenhall vers votre propre bloc de code. Sinon, vous n'obtiendrez pas exactement les résultats présentés ici.
```
Comment (mariechristineb) [inline fr/lecons/introduction-a-la-stylometrie-avec-python.md]: petite coquille dans 'Kilgariff'

Comment (fdlaramee) [reply fr/lecons/introduction-a-la-stylometrie-avec-python.md]: Merci @mariechristineb c'est réglé.

**Review by mariechristineb — APPROVED** [review]

**Pull request merged**